terraform/environments/ops/variables.tf

```diff
@@ -9,10 +9,10 @@ variable "region" {
   description = ""
 }
 
-variable "deploy_triggers" {
+variable "cd_system" {
```

Comment:
`deploy` feels like a verb here - can we call this `setup_cd_system` or something like that?

Reply:
fixed